contrib/mesos/pkg/executor/service/service.go

```diff
@@ -70,6 +71,24 @@ type KubeletExecutorServer struct {
 	SuicideTimeout time.Duration
 	ShutdownFD     int
 	ShutdownFIFO   string
+	CgroupRoot     string
+}
+
+func findMesosCgroup() string {
+	// derive our cgroup from MESOS_DIRECTORY environment
+	mesosDir := os.Getenv("MESOS_DIRECTORY")
+	if mesosDir == "" {
+		log.V(2).Infof("cannot derive executor's cgroup because MESOS_DIRECTORY is empty")
+		return ""
+	}
+
+	containerId := path.Base(mesosDir)
+	if containerId == "" {
+		log.V(2).Infof(`cannot derive executor's cgroup from MESOS_DIRECTORY="%v"`, mesosDir)
+		return ""
+	}
+	cgroupRoot := fmt.Sprintf("/mesos/%v", containerId)
```

Comment:
+1 for configurability, perhaps also via env var? defaulting to /mesos is fine. would be nice to see "/mesos" as a constant as well.